src/diffusers/pipelines/stable_diffusion/safety_checker.py

```diff
@@ -33,6 +33,10 @@ def __init__(self, config: CLIPConfig):
 
     @torch.no_grad()
     def forward(self, clip_input, images):
+        is_input_numpy = isinstance(images, np.ndarray)
+        if is_input_numpy:
```

Comment:
Let's try to avoid one-liners -> `is_input_numpy` is not easier to understand or more readable than `isinstance(images, np.ndarray)` IMO, but it requires one to have to remember one more attribute.

```suggestion
        if isinstance(images, np.ndarray):
```

Reply:
we need to remember the type to convert the outputs back, so `is_input_numpy` this is used later in the function